using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -862,271 +1131,153 @@ var targetParameters = new ACPTargetParameters(mboxParameters, profileParameters
++ (void) setPreviewRestartDeeplink: (nonnull NSURL*) deeplink;
 ```
 
+* _deeplink_ is a NSURL containing the preview restart deeplink.
+
 #### Examples
 
 Here are some examples in Objective-C and Swift:
 
 **Objective-C**
 
-```objectivec
-ACPTargetProduct *product = [ACPTargetProduct targetProductWithId:@"24D334" categoryId:@"Stationary"];
-
-ACPTargetOrder *order = [ACPTargetOrder targetOrderWithId:@"ADCKKBC" total:@(400.50) purchasedProductIds:@[@"34", @"125"]];
-
-ACPTargetParameters *targetParameters = [ACPTargetParameters targetParametersWithParameters:nil
-profileParameters:nil
-product:product
-order:order];
-
-[ACPTarget locationsDisplayed:@[@"mboxName1", @"mboxName2"] withTargetParameters:targetParameters];
+```text
+[ACPTarget setPreviewRestartDeepLink:@"myapp://HomePage"];
 ```
 
 **Swift**
 
 ```swift
-let product = ACPTargetProduct(id: "24D334", categoryId: "Stationary")
+ACPTarget.setPreviewRestartDeepLink("myapp://HomePage")
+```
+{% endtab %}
 
-let order = ACPTargetOrder(id: "ADCKKBC", total: NSNumber(value: 400.50), purchasedProductIds: ["34", "125"])
+{% tab title="React Native" %}
 
-let targetParameters = ACPTargetParameters(parameters: nil, profileParameters: nil, product: product, order: order)
+#### Syntax
 
-ACPTarget.locationsDisplayed(["mboxName1", "mboxName2"], with: targetParameters)
+```javascript
+setPreviewRestartDeeplink(deepLink: string)
 ```
-{% endtab %}
+* _deepLink_ is a string containing the preview restart deeplink.
 
-{% tab title="React Native" %}
-### locationsDisplayed
+#### Examples
 
 **JavaScript**
 
 ```javascript
-var product = new ACPTargetProduct("24D334", "Stationary");
-var order = new ACPTargetOrder("ADCKKBC", 400.50, ["34", "125"]);
-var targetParameters = new ACPTargetParameters(null, null, product, order);
-
-ACPTarget.locationsDisplayed(["mboxName1", "mboxName2"], targetParameters);
+ACPTarget.setPreviewRestartDeeplink("myapp://HomePage");
 ```
+
 {% endtab %}
-{% endtabs %}
 
-## Send an mbox click notification
+{% endtabs %}
 
-Use this API to send a location \(mbox\) click notification to configured Target server. This API can be invoked in the following cases:
+## setThirdPartyId
 
-* For a prefetched mbox, after the mbox content is retrieved using [retrieveLocationContent API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#retrieve-location-content-requests).
-* For a regular mbox when no previous prefetch request is made and the mbox content is retrieved using `retrieveLocationContent` API.
+This API sets the custom visitor ID for Target.
 
-{% hint style="warning" %}
-For a click notification to be sent to Target, make sure click metric is enabled for the given mbox name in Target.
+{% hint style="info" %}
+This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall or when the `resetExperience` API is used.
```

Comment:
should the `resetExperience` API be linked to?

Reply:
@rymorale, I don't think so.